assets/sass/cds/_sections.scss

```diff
@@ -1469,3 +1407,76 @@ article.post {
     margin-bottom: 0;
   }
 }
+
+.work-with-us--apply-now {
```

Comment:
Can you move this block back up to where it was to make it easier to see the difference?

Reply:
Done